### PR TITLE
Sparse field lists and specified related resources

### PR DESCRIPTION
**Pull request by 21echoes**
Make it clear how to use JSON:API to customize the data you receive from our API

### CODE REVIEW

**Comment by joshbuddy** [comment]
This looks reasonable to me!

File: patreon/schemas/campaign.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+class Attributes(object):
```
Comment (LiraNuna) [inline patreon/schemas/campaign.py]: What's your plan on keeping this up to date?

Comment (21echoes) [inline patreon/schemas/campaign.py]: first thought: deploying a new schema on patreon.com kicks off a script which opens a PR against this repo. seemed wayyy out of scope for this PR, but definitely something to think about.

Comment (21echoes) [inline patreon/schemas/campaign.py]: added as an issue here: https://github.com/Patreon/patreon-python/issues/5

**Pull request merged**